documentation/vespa-quick-start-multinode-aws.html

```diff
@@ -43,91 +43,47 @@ <h2 id="launch-instances">Launch instances on <a href="aws.amazon.com/ec2">AWS E
 <h2 id="install-vespa">Install and run Vespa</h2>
 
 <p> Create a hostfile listing the hosts in the system
-(<em>vespa-instances.txt</em>) to simplify the process multinode
-install process.
+(<samp>hosts.txt</samp>) to simplify the multinode install process.
 </p>
 <pre class="brush: cli">
-$ cat vespa-instances.txt
+$ cat hosts.txt
 ip-172-1-1-1.us-east-2.compute.internal
```

Comment:
It is not obvious for me what these names are and where to find them. I think we should mention that these are the private dns addresses for the EC2 instances.